client/src/components/Notification.js

```diff
@@ -12,22 +12,89 @@ const readableTimeAgo = (time) => {
   return daysAgo > 0 ? `${daysAgo} days ago` : 'today'
 }
 
-export const Notification = ({ notification, Link = NextLink }) => {
+const getLink = (linkType, label, notification) => {
+  if (linkType === 'request') {
+    const href = `/account/requests/${notification.material_request.id}`
+    return (
+      <Link href={href} key={1}>
+        <Anchor href="#" label={label || 'request'} />
+      </Link>
+    )
+  }
+  if (linkType === 'material_name') {
+    const href = `/resources/${notification.material.id}`
+    return (
+      <Link href={href} key={2}>
+        <Anchor href="#" label={label || notification.material.title} />
```

Comment:
can we set the href of the Anchor to match the link so if a user wants to open in a new tab via their browser it will work?

Reply:
Yep! Done